packages/devtools_app/lib/src/memory/flutter/memory_heap_tree_view.dart

```diff
@@ -537,6 +620,97 @@ class HeapTreeViewState extends State<HeapTree> with AutoDisposeMixin {
     );
   }
 
+  bool _classMatcher(HeapGraphClassLive liveClass) {
+    final regExp = RegExp(knownClassesRegExs);
+    return regExp.allMatches(liveClass.name).isNotEmpty;
+  }
+
+  List<ExternalReference> _collect(List dupSearch) {
+    final externalsToAnalyze = <ExternalReference>[];
+
+    // Analyze the external heap for memory information
+    final root = controller.libraryRoot;
+    assert(root != null);
+
+    print(">>>> RegEx = $knownClassesRegExs");
+
+// TODO(terry): Remove dupSearch debug code.
+    for (final library in root.children) {
+      if (library.isExternals) {
+        final ExternalReferences externals = library;
+        for (final ExternalReference external in externals.children) {
+          assert(external.isExternal);
+
+          final liveExternal = external.liveExternal;
+          final size = liveExternal.externalProperty.externalSize;
+          final liveElement = liveExternal.live;
+          final HeapGraphClassLive liveClass = liveElement.theClass;
+          if (_classMatcher(liveClass)) {
+            externalsToAnalyze.add(external);
+          }
+
+          // TODO(terrry): Remove debug code.
```

Comment:
is this debug code or did you remove the debug code?

Reply:
It was removed.